iml-warp-drive/src/cache.rs

```diff
@@ -147,6 +147,12 @@ pub async fn db_record_to_change_record(
                 Ok(RecordChange::Update(Record::LnetConfiguration(x)))
             }
         },
+        DbRecord::ContentType(x) => match (msg_type, x) {
+            (MessageType::Delete, x) => Ok(RecordChange::Delete(RecordId::ContentType(x.id()))),
+            (MessageType::Insert, x) | (MessageType::Update, x) => {
+                Ok(RecordChange::Update(Record::ContentType(x)))
+            }
+        },
```

Comment:
Looks like a part of https://github.com/whamcloud/integrated-manager-for-lustre/pull/1574?

Reply:
Yeah, I need to build out a patch stack until this and other work lands. I've updated the base branch of #1574 to point to this patch so those files will fall out of that PR.